truffle/src/main/ruby/core/process_mirror.rb

```diff
@@ -109,6 +110,7 @@ def initialize(env_or_cmd, *args)
 
             @command = command
             @argv = argv
+            @env_array = []
```

Comment:
Thanks. This was leftover from when I used to bail early if `@options` wasn't defined. Setting that in the constructor fixes everything.